.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: ci
+on:
+  pull_request:
+  push:
+    branches: [master]
```

Comment:
We won't actually see this run on my fork until this gets merged down into the base repository, which I'm realizing is not helpful. I can update this to run on _all_ `push` events for demonstration purposes.

Reply:
That actually still didn't work, but see [this example](https://github.com/namoscato/vue-gapi/runs/770214140) in a pull request within my fork. Still looking into why things are failing at the moment. I think we might be up against a lot of outdated dependencies.